paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java

```diff
@@ -71,20 +71,19 @@ public List<SchemaVO> getSchema(MetadataDTO dto) {
                     internalRow -> {
                         SchemaVO schemaVo =
                                 SchemaVO.builder()
-                                        .setSchemaId(internalRow.getLong(0))
-                                        .setFields(
+                                        .schemaId(internalRow.getLong(0))
+                                        .fields(
                                                 new Gson()
                                                         .fromJson(
                                                                 internalRow.getString(1).toString(),
                                                                 new TypeToken<
                                                                         LinkedList<
                                                                                 MetadataFieldsModel>>() {}))
-                                        .setPartitionKeys(internalRow.getString(2).toString())
-                                        .setPrimaryKeys(internalRow.getString(3).toString())
-                                        .setOption(
-                                                formatOptions(internalRow.getString(4).toString()))
-                                        .setComment(internalRow.getString(5).toString())
-                                        .setUpdateTime(
+                                        .partitionKeys(internalRow.getString(2).toString())
```

Comment:
Could this cause `NullPointerException` when `internalRow.getString(2)` is null?